test/system/090-events.bats

```diff
@@ -25,3 +25,35 @@ load helpers
     run_podman events --filter type=container --filter container=$cname --filter event=start --stream=false
     is "$output" "$expect" "filtering just by label"
 }
+
+@test "image events" {
+    # `remove events` seem to be broken...
+    skip_if_remote "FIXME: -remote is missing many events"
+    pushedDir=$PODMAN_TMPDIR/dir
+    mkdir -p $pushedDir
+
+    tarball=$PODMAN_TMPDIR/ball.tar
+
+    run_podman image inspect --format "{{.ID}}" $IMAGE
+    imageID="$output"
+
+    tag=registry.com/$(random_string 10 | tr A-Z a-z)
+
+    run_podman push $IMAGE dir:$pushedDir
+    run_podman save $IMAGE -o $tarball
+    run_podman load -i $tarball
+    run_podman pull docker-archive:$tarball
+    run_podman tag $IMAGE $tag
+    run_podman untag $IMAGE $tag
+    run_podman tag $IMAGE $tag
+    run_podman rmi $tag
+
+    run_podman events --stream=false --filter type=image
+    is "$output" ".*image push.* $imageID dir:$pushedDir.*" \
+        ".*image save.* $imageID $tarball.*" \
+        ".*image loadfromarchive.* $tarball.*" \
+        ".*image pull.* docker-archive:$tarball.*" \
+        ".*image tag.* $imageID $tag.*" \
+        ".*image untag.* $imageID $tag.*" \
+        ".*image remove.* $imageID $tag.*"
```

Comment:
This does not do what you think it does (and, sadly, I was not clever enough to add an error check to `is()` such that it would catch this). What I think you want is:
```bash
   is "$output" ".*image push $imageID dir:$pushedDir
.*image save $imageID $tarball
.*image loadfromarchive *$tarball
.*image pull *docker-archive:$tarball
.*image tag $imageID $tag
.*image untag $imageID $tag:latest
.*image tag $imageID $tag
.*image remove $imageID $tag.*" \
       "podman events"
```
(please copy-paste from the above, because I made a lot of changes including adding a missing event, removing a lot of unwanted `.*`s, and adding a missing `:latest`).

Reply:
Thanks a lot, Ed!